docs/dev/devel-env.rst

```diff
@@ -127,9 +127,25 @@ In case you don't have a Last.FM API key, you can get it from `Last.FM API page`
 
 You also need to update the ``API_URL`` field value to ``http://localhost``.
 
+We also have a Spotify importer script which imports listens from
+Spotify automatically using the Spotify API. In order to run this in your
+local development environment, you'll have to register an application on the
+`Spotify Developer Dashboard`_. Use ``http://localhost/profile/connect-spotify/callback``
+as the callback URL.
```

Comment:
Something about how to change the config file to change the callback url if you want/need the hostname to be different?

"The hostname on this callback URL must be the same as the host that you access your development server from. If this is different then you should also change SPOTIFY_CALLBACK_HOST blah blah blah"